### ISSUE TITLE
Test setup fails when using chain built with Cosmos SDK v0.46.0

### ISSUE THREAD

**Issue by chatton** [strangelove-ventures/ibctest#232]
When running a test using Cosmos SDK `v0.46.0`, the `facet` accounts fail to be initialised.

Sample output from failed test.

```
=== RUN   TestFeeMiddlewareTestSuite
=== RUN   TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender
{"level":"info","msg":"Running command","command":"rly config init --home /var/relayer-rly","container":"rly-TestFeeMiddlewareTestSuite_TestMultiMsg_MsgPayPacketFeeSingleSender-NewDockerRelayer-fkd"}
{"level":"debug","msg":"stdout:\n\nstderr:\n","container":"rly-TestFeeMiddlewareTestSuite_TestMultiMsg_MsgPayPacketFeeSingleSender-NewDockerRelayer-fkd"}
    logger.go:130: 2022-08-08T09:42:55.224+0100	INFO	About to start container	{"chain_id": "chain-b", "test": "TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender", "image": "ghcr.io/cosmos/ibc-go-simd-e2e:main", "test_name": "TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender", "command": "simd keys add faucet --keyring-backend test --output json --home /tmp/simapp-b", "hostname": "TestFeeMiddlewareTestSuite_Tes_._ayPacketFeeSingleSender-ymfvxd", "container": "TestFeeMiddlewareTestSuite_TestMultiMsg_MsgPayPacketFeeSingleSender-ymfvxd"}
    logger.go:130: 2022-08-08T09:42:55.224+0100	INFO	About to start container	{"chain_id": "chain-a", "test": "TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender", "image": "ghcr.io/cosmos/ibc-go-simd-e2e:main", "test_name": "TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender", "command": "simd keys add faucet --keyring-backend test --output json --home /tmp/simapp-a", "hostname": "TestFeeMiddlewareTestSuite_Tes_._ayPacketFeeSingleSender-rfnygp", "container": "TestFeeMiddlewareTestSuite_TestMultiMsg_MsgPayPacketFeeSingleSender-rfnygp"}
    testsuite.go:120:
        	Error Trace:	/Users/chatton/checkouts/ibc-go/e2e/testsuite.go:120
        	            				/Users/chatton/checkouts/ibc-go/e2e/fee_middleware_test.go:205
        	Error:      	Received unexpected error:
        	            	failed to create faucet accounts: failed to create common account with name faucet: failed to get account address for key "faucet" on chain simapp-a: Bytes left over in UnmarshalBinaryLengthPrefixed, should read 10 more bytes but have 153
        	Test:       	TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender
    setup.go:132: Pruned unused networks: [ibctest-sxzotifn]
--- FAIL: TestFeeMiddlewareTestSuite (3.93s)
    --- FAIL: TestFeeMiddlewareTestSuite/TestMultiMsg_MsgPayPacketFeeSingleSender (3.93s)
```

When running the same test using an image built using `v0.45.6` (same as ibctest) everything works as expected.

We've upgraded to `v0.46.0` on `main` in `ibc-go` and as a result we can no longer run e2e tests.

**Comment by chatton** [comment]
It looks like maybe this will be resolved by #209

**Comment by chatton** [comment]
Update to v0.46 has already been merged, closing this.
